guides/assets/javascripts/turbo.js

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
How did you generate this file?
It seems a bit different from: https://unpkg.com/browse/@hotwired/turbo@7.1.0/dist/turbo.es2017-esm.js

Reply:
I took the 7.1.0 umd version and ran it through a minimizer: https://unpkg.com/browse/@hotwired/turbo@7.1.0/dist/turbo.es2017-umd.js

I'm happy to replace it with the regular version, just thought I keep it minified because the previous one was too.

Reply:
Could you post how you minified it? This makes it easier to review and reproduce in case we need to upgrade.

Reply:
In this case I just used the [built-in one in DuckDuckGo](https://duckduckgo.com/?q=js+minifier&ia=answer). Though something like [`minify`](https://github.com/tdewolff/minify/tree/master/cmd/minify) could also do the trick locally.

But maybe we are just better off by including the unminified version so it's easier to compare and verify.

Reply:
> But maybe we are just better off by including the unminified version so it's easier to compare and verify.

I think this is a good idea. It's not that big a script, and this way readers can more easily see Turbo in action via devtools if they're interested.

Reply:
> I think this is a good idea. It's not that big a script, and this way readers can more easily see Turbo in action via devtools if they're interested.

Agreed. I replaced the script with the unminified version and put the source url into the top of the file.

Reply:
Ok, let's stick with the unminified version 👍